src/OpenTelemetry/Metrics/MetricReaderExt.cs

```diff
@@ -156,8 +156,8 @@ internal List<Metric> AddMetricsListWithViews(Instrument instrument, List<Metric
                     }
                     else
                     {
-                        Metric metric;
-                        metric = new Metric(metricStreamIdentity, this.GetAggregationTemporality(metricStreamIdentity.InstrumentType), this.maxMetricPointsPerMetricStream, metricStreamIdentity.HistogramBucketBounds, metricStreamIdentity.TagKeys);
+                        bool histogramRecordMinMax = (metricStreamConfig as HistogramConfiguration)?.RecordMinMax ?? true;
+                        Metric metric = new(metricStreamIdentity, this.GetAggregationTemporality(metricStreamIdentity.InstrumentType), this.maxMetricPointsPerMetricStream, metricStreamIdentity.HistogramBucketBounds, metricStreamIdentity.TagKeys, histogramRecordMinMax);
```

Comment:
@alanwest Could you take a close look at this portion? I think the bool recordMinMax should be part of the MetricIdentity.

Reply:
Yes, for sure. So something like the following should produce the duplicate metric instrument warning above

```csharp
            using var meter = new Meter(Utils.GetCurrentMethodName());
            var histogram = meter.CreateHistogram<double>("MyHistogram");
            var exportedItems = new List<Metric>();
            using var meterProvider = Sdk.CreateMeterProviderBuilder()
                .AddMeter(meter.Name)
                .AddView(histogram.Name, histogramConfigurationWithMinMax)
                .AddView(histogram.Name, histogramConfigurationWithoutMinMax)
                .AddInMemoryExporter(exportedItems)
                .Build();
```

That is each view will produce a metric stream which will be duplicated by name.

Reply:
Moved `HistogramRecordMinMax` into `MetricStreamIdentity`, let me know if that looks good